blocks/filter-group/filter-group.js

```diff
@@ -0,0 +1,40 @@
+export default async function decorate(block) {
+    const label = document.createElement('label');
```

Comment:
this shouldn't be a label element, because these buttons aren't inputs. let's use a div or span element with an `aria-describedby` on the filter group to better support screen readers. good thinking, just need a little bit of a tweak.